### PR TITLE
Fix loading data from bag file.

### PR DESCRIPTION
**Pull request by malban**
This fixes the CI unit tests (for https://github.com/IntelRealSense/realsense-ros/pull/1504)

### CODE REVIEW

**Review by 130s — APPROVED** [review]
Looks like something I happened to delete while resolving the conflict that happened when synching the branch to upstream? :bowing_man: 

Merge when CI passes.

**Comment by 130s** [comment]
Interestingly, Bionic job passed but Xenial and Focal jobs failed https://travis-ci.org/github/plusone-robotics/realsense/builds/744729404

**Comment by malban** [comment]
> Interestingly, Bionic job passed but Xenial and Focal jobs failed

Strange.  All the tests but one are passing, but it's a different one for xenial and focal.  Maybe there is a timing issue.  The only thing I can think of is that there is now a 1 second delay at startup of the driver.

File: realsense2_camera/src/realsense_node_factory.cpp [hunk 1]
```diff
@@ -270,7 +270,7 @@ void RealSenseNodeFactory::onInit()
 	auto privateNh = getPrivateNodeHandle();
 
 	privateNh.param("initial_reset", _initial_reset, false);
-	_init_timer = nh.createWallTimer(ros::WallDuration(1.0), &RealSenseNodeFactory::initialize, this, true);
+	_init_timer = nh.createWallTimer(ros::WallDuration(0.1), &RealSenseNodeFactory::initialize, this, true);
```
Comment (130s) [inline realsense2_camera/src/realsense_node_factory.cpp:273]: IIUC this changes the timer's callback from 1 sec to 0.1 sec? No idea if that's acceptable in the upstream (but maybe you have a rationale @malban). Let's hear what the maintainers say.

**Review by 130s — APPROVED** [review]
All jobs passed :champagne: 

This makes the upstream PR ready for re-review. Thanks.

File: realsense2_camera/src/realsense_node_factory.cpp (hunk 1 above)
Comment (malban) [reply realsense2_camera/src/realsense_node_factory.cpp:273]: This timer is new, so I don't thing the change in duration will matter to upstream.  The duration isn't important.  It's just to separate the driver initialization from the onInit() function, which shouldn't block or do significant work: http://wiki.ros.org/nodelet#onInit

In my experience, if onInit takes too long, nodelets can silently fail to correctly advertise/subscribe to topics.

**Pull request merged**